views/profile.handlebars

```diff
@@ -0,0 +1,11 @@
+<class class="profile-container">
+    <section class="header">
+        <h3 class="author">{{author}}</h3>
+    </section>
+
+    <section class="content">
+        <p class="twitch">{{twitch_link}}</p>
+        <p class="twitch">{{youtube_link}}</p>
+        <p class="twitch">{{bio}}</p>
+    </section>
+</class>
```

Comment:
It's best practice to leave a empty line at the end of a file. This is for formatting issues that can arise when automated scripts merge files and such, although that's not likely to happen with us, Github has annoying reminders.